src/main/java/kr/co/antoon/oauth/dto/Token.java

```diff
@@ -0,0 +1,18 @@
+package kr.co.antoon.oauth.dto;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@ToString
+@NoArgsConstructor
+@Getter
+public class Token {
```

Comment:
```suggestion
public class TokenDTO {
```

동건님 말씀대로 명확한 명칭 dto 패키지에 포함되더라도 실제 코드에서 보면 DTO인지 구분이 안갑니다.